go/protocol/chat1/extras.go

```diff
@@ -75,7 +76,7 @@ func (cid ConversationID) DbShortForm() ConvIDShort {
 }
 
 func (cid ConversationID) DbShortFormString() string {
-	return hex.EncodeToString(cid.DbShortForm())
+	return DbShortFormToString(cid.DbShortForm())
```

Comment:
What is this?

Reply:
nit fix -- calling the helper defined at L83 instead of calling ` hex.EncodeToString(` again